li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumer.java

```diff
@@ -394,6 +513,23 @@
   @InterfaceOrigin.ApacheKafka
   long position(TopicPartition partition);
```

Comment:
Does this still exist in the open source client?

Reply:
Yes. https://github.com/apache/kafka/blob/2.0.1/clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java#L1592